### ISSUE TITLE
Add some description / explanation when request cart is empty

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/pulfalight#1746]
refs #1717 
[full list of issues for audit1717](https://github.com/pulibrary/pulfalight/issues?q=is%3Aissue%20state%3Aopen%20audit1717)

### Location of problem on site

Request cart

### Description of error / warning

When the cart is empty and I'm navigating the site with VoiceOver, it announces that there is a table but then you can't navigate into the table the way it says to. It sort of feels like the data is missing or not structured correctly / inaccessible.

### WCAG success criterion

[1.3.1 Info and Relationships](https://www.w3.org/TR/WCAG22/#info-and-relationships)

### WCAG conformance level

A

### Recommended fix

Maybe don't show the table if it doesn't have content? Maybe show some message about what will be there when an item is added?

### Tool used to find the problem

VoiceOver
